src/syncer.ts

```diff
@@ -448,6 +452,15 @@ export class Syncer {
     const gitRemote = `https://x-access-token:${token}@github.com/${org}/${repoName}.git`;
     const upstreamGitRemote = `https://github.com/${upstreamOrg}/${repoName}.git`;
 
+    // Get the owners to parse the most recent commit author that is also an owner
```

Comment:
It seems you collect all the assignees so maybe this comment needs to be updated?

Reply:
Ah, good catch. Yeah, this was the initial approach but in the end figured more attention on it was better.